assets/js/blocks/product-gallery/inner-blocks/product-gallery-pager/index.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { isExperimentalBuild } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import { ProductGalleryBlockIcon } from './icons';
+import { Edit, Save } from './edit';
+import metadata from './block.json';
+import './style.scss';
+import './editor.scss';
+
+if ( isExperimentalBuild() ) {
+	// @ts-expect-error: `metadata` currently does not have a type definition in WordPress core
+	registerBlockType( metadata, {
+		icon: ProductGalleryBlockIcon,
```

Comment:
I'd change the name of the icon to anything but `ProductGalleryBlockIcon`. Currently, it's confusing because it's actually the Pager block icon and not the Product Gallery block icon.

I'd stick to the icons name of the other inner blocks: `Icon`

Reply:
Nice catch! I renamed to `ProductGalleryPagerBlockIcon`, since we are using named exports, I think the name should be more descriptive than only `Icon`, it also helps the intellisense.